core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5146,6 +5146,12 @@ components:
           minimum: 0.01
           maximum: 9999999999.99
           multipleOf: 0.01
+        deleteEntityType:
+          type: string
+          description: Type of root entity to delete after submission.  If it matches the submission entity, will delete intermediate set entity created for submission.
+        deleteEntityName:
+          type: string
+          description: Name of root entity to delete after submission.  If it matches the submission entity, will delete intermediate set entity created for submission..
```

Comment:
the tiniest nit: `deleteEntityName` description ends with two periods

also, totally optional and your call: you _could_ make this a single property delimited by e.g. a slash. Since slash is not a legal character in either entity types or names, it's safe to use as a delimiter.